node-src/git/getChangedFilesWithReplacement.test.ts

```diff
@@ -125,4 +126,153 @@ describe('getChangedFilesWithReplacements', () => {
       })
     ).rejects.toThrow(/fatal: bad object missing/);
   });
+
+  it('tries a replacement builds until it finds a working one', async () => {
```

Comment:
These tests pass even when `node-src/git/getChangedFilesWithReplacement.ts` is on `main`. Is there a valid test that fails on `main` but passes on this branch?